code/go/0chain.net/smartcontract/minersc/models.go

```diff
@@ -821,6 +821,9 @@ type SimpleNode struct {
 
 	// LastHealthCheck used to check for active node
 	LastHealthCheck common.Timestamp `json:"last_health_check"`
+
```

Comment:
I don't think we need this. Instead emit en event in [createMagicBlock](https://github.com/0chain/0chain/blob/1e8779f639d24bcd832baaf07adbd2e31915b066/code/go/0chain.net/smartcontract/minersc/dkg.go#L798) with the `Status` update converting to a boolean Active.